translations/en-us.yaml

```diff
@@ -194,7 +193,7 @@ general:
   noSessionLearningMaterialsAvailable: "No Session Learning Materials Available"
   notAClerkship: "Not a Clerkship"
   notes: Notes
-  notFoundMessage: "Rats! I couldn't find that. Please check your page address, and try again."
+  notFoundMessage: "Rats! I couldn''t find that. Please check your page address, and try again."
```

Comment:
😡

Reply:
yep.

Reply:
turns out - that's also not necessary. so the escape character doesn't have to be escaped itself. that's a first.